src/Extension/BetterNavigatorExtension.php

```diff
@@ -8,25 +8,44 @@
 use SilverStripe\Control\Director;
 use SilverStripe\Core\Config\Config;
 use SilverStripe\ORM\DataExtension;
+use SilverStripe\ORM\FieldType\DBField;
 use SilverStripe\Security\LogoutForm;
 use SilverStripe\Security\Member;
 use SilverStripe\Security\Permission;
 use SilverStripe\Security\Security;
 use SilverStripe\Versioned\Versioned;
 use SilverStripe\View\ArrayData;
+use SilverStripe\ORM\FieldType\DBHTMLText;
 
-class BetterNavigatorExtension extends DataExtension {
+class BetterNavigatorExtension extends DataExtension
+{
+
+    /**
+     * @var string
+     */
+    private $navigatorHTML;
+
+    /**
+     * Noop, pseudo backward compatability only
+     * @return DBHTMLText
+     */
+    public function BetterNavigator()
```

Comment:
Please chop this out, I'll do a major release

Reply:
done